src/main.cpp

```diff
@@ -3401,7 +3509,14 @@ bool CInputStreamAdaptive::Open(const kodi::addon::InputstreamProperty& props)
                 max_user_bandwidth);
     }
     else if (prop.first == "inputstream.adaptive.play_timeshift_buffer")
+    {
       m_playTimeshiftBuffer = stricmp(prop.second.c_str(), "true") == 0;
+    }
+    else if (prop.first == "inputstream.adaptive.pre_init_data")
+    {
+      kodi::Log(ADDON_LOG_DEBUG, "found inputstream.adaptive.pre_init_data: [not shown]");
```

Comment:
We really need to explain here what pre_init_data means. Is this documented somewhere?

Reply:
in the wiki i can document it when it is merged,
I can add a short comment line there as well

Reply:
done (in the source code)